docs/webhooks.rst

```diff
@@ -109,7 +108,7 @@ branches
 
 token
     The integration token. You'll find this value on the project's
-    **Integrations** Admin dashboard page.
+    :guilabel:`Admin` > :guilabel:`Integrations` dashboard page.
```

Comment:
Same here, above sentence should be changed.

Reply:
The integration token found on the project's **Integrations**  dashboard page. (:guilabel:`Admin` > :guilabel:`Integrations`)